README.md

```diff
@@ -1,3 +1,69 @@
+## What is Open Source?
+
+Open source refers to software or projects whose source code is made available to the public, allowing anyone to view, use, modify, and distribute the code. This collaborative approach fosters a vibrant community of developers working together to improve and create amazing projects.
+
+## Getting Started
+
+### Step 1: Find an Open Source Project
+
+To get started, find an open source project that aligns with your interests and skills. Look for projects that welcome beginner contributions and have clear guidelines for newcomers.
+
+### Step 2: Understand the Project
+
+Take some time to understand the project's purpose, goals, and existing codebase. Read the project's documentation and get familiar with its structure.
+
+### Step 3: Set Up Your Development Environment
+
+Follow the project's instructions to set up your development environment. This may involve installing Git, Node.js, and other dependencies.
+
+### Step 4: Create a GitHub Account
+
+Most open source projects use GitHub for collaboration. Create a GitHub account if you don't have one already.
+
+## Contributing to cloudnative.nz 
+
+### Step 5: Downloading the source code
+For this site follow these steps to start contributing:
+
+- Create a github account 
+- Fork this repo 
+- Run `git clone https://github.com/<yourusername>/site.git` in your command line to pull down the forked repo to your device 
+- Run this website by opening the index.html file from your file explorer
+- View the issues and decide on which one to work through, ask questions or ask for assistance in the thread of the issue if you need to.
+
+
+### Step 6: Submit a Pull Request
+
+When your changes are ready, submit a pull request to the original project repository. A pull request is a request to merge your changes into the main codebase. Be sure to follow the project's contribution guidelines and provide a clear description of your changes.
+
+### Step 7: Engage with the Community
+
+Engage with the community by participating in discussions, commenting on issues, and helping other contributors. Collaboration is a key aspect of open source, and being an active community member enhances your open source experience.
+
+## Adding New Content
+
+### Step 9: Create New Pages or Sections
+
+Do you have ideas for new sections or pages for the website? Whether it's a new tutorial, a glossary, or an FAQ section, you can propose these additions to the project maintainers. Open an issue on the repository to discuss your ideas and get feedback from the community.
+
+### Step 10: Implement Your Changes
+
+If your ideas are well-received, implement the changes by adding new pages or sections to the website. Follow the project's code and style guidelines to ensure consistency.
+
+### Step 11: Submit Another Pull Request
+
+Once your changes are ready, submit another pull request with the new content. Remember to document your additions and explain their purpose in the pull request description.
+
```

Comment:
Remeber to [link you issue to the pull requiest](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) to help everyone keep track of the work.